.github/kokoro/install-200t.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copied from install.sh
+# Only difference is INSTALL_DEVICE in CMAKE_FLAGS and tarball name
+
+SCRIPT_SRC="$(realpath ${BASH_SOURCE[0]})"
+SCRIPT_DIR="$(dirname "${SCRIPT_SRC}")"
+INSTALL_DIR="$(pwd)/github/${KOKORO_DIR}/install"
+
+export CMAKE_FLAGS="-GNinja -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -DINSTALL_DEVICE=xc7a200t"
+export BUILD_TOOL=ninja
+source ${SCRIPT_DIR}/common.sh
+
+echo
+echo "========================================"
+echo "Running install tests (make install)"
+echo "----------------------------------------"
+(
+	source env/conda/bin/activate symbiflow_arch_def_base
+	pushd build
+	export VPR_NUM_WORKERS=${CORES}
+	ninja -j${MAX_CORES} install
+	popd
+)
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Compressing and uploading install dir"
+echo "----------------------------------------"
+(
+	du -ah install
+	pushd install
+	export GIT_HASH=$(git rev-parse --short HEAD)
+	tar vcf - * | xz -9 -T${MAX_CORES} - > ../symbiflow-arch-defs-install-200t-${GIT_HASH}.tar.xz
+	popd
+)
+echo "----------------------------------------"
+
+echo
+echo "========================================"
+echo "Running installed toolchain tests"
+echo "----------------------------------------"
+echo "Skipped on single architecture package."
+# TODO adapt this test
```

Comment:
You need to create a new test for this.

Reply:
This class of test literally just caught a problem with https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1640, so it is delivering value.